src/backend/Aggregator/Domains/DataTransferObjects/Photos/PhotoListDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Domains.DataTransferObjects.Photos
+{
+    public class PhotoListDTO
+    {
+        public Guid Id { get; set; }
+        public DateTime UploadDate { get; set; }
+
+        public string Name { get; set; }
+        public string Description { get; set; }
+
+        public string PhotoUrl64 { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this property type to 'System.Uri'.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=5020178)